backend/app/repositories/chat.py

```diff
@@ -220,6 +220,8 @@ def chat_trend_by_origin(
 
         stats.sort(key=lambda x: x["date"])
         return stats
-
+    
+    def list_chat_origins(self, session: Session):
+        return session.exec(select(Chat.origin, Chat.id).order_by(Chat.created_at.desc()))
```

Comment:
The value of the `origin` field in the `chats` table may be repeated, please use `distinct`.